### PR TITLE
parity: 2.5.6 -> 2.5.7, parity-beta: 2.6.1 -> 2.6.2

### PR DESCRIPTION
**Pull request by xrelkd**
###### Motivation for this change
Update `parity` and `parity-beta` to the latest releases.
- https://github.com/paritytech/parity-ethereum/releases/tag/v2.5.7
- https://github.com/paritytech/parity-ethereum/releases/tag/v2.6.2
- https://www.parity.io/parity-ethereum-security-release-29-8-19/

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @akru

### CODE REVIEW

**Review by akru — APPROVED** [review]
Thank you for your work!

**Comment by xrelkd** [comment]
@akru You're welcome!
Hi, @worldofpeace @marsam could you review this PR? Thank you very much!

File: pkgs/applications/blockchains/parity/parity.nix [hunk 1]
```diff
@@ -26,10 +26,7 @@ rustPlatform.buildRustPackage rec {
     inherit sha256;
   };
 
-  buildInputs = [
-    pkgconfig cmake perl
-    systemd.lib systemd.dev openssl openssl.dev
-  ];
+  buildInputs = [ cmake openssl perl pkgconfig systemd ];
```
Comment (worldofpeace) [inline pkgs/applications/blockchains/parity/parity.nix]: Are we sure we need `cmake` and `pkgconfig` as `buildInputs`? Those are typically only needed during the build so they're `nativeBuildInputs`.

Comment (worldofpeace) [reply pkgs/applications/blockchains/parity/parity.nix]: Also where is perl needed https://github.com/paritytech/parity-ethereum/search?q=perl&type=Code ?

**Comment by xrelkd** [comment]
@worldofpeace Thanks for the review!
`buildInputs` and `nativeBuildInputs` have been updated.

@GrahamcOfBorg build parity parity-beta

**Comment by xrelkd** [comment]
Hi @worldofpeace, is there any other improvement I can do for this PR?

**Comment by worldofpeace** [comment]
> Hi @worldofpeace, is there any other improvement I can do for this PR?

Seeing if it can be integrated now. Thanks for the reminder.

**Review by worldofpeace — APPROVED** [review]
LGTM. Still builds.

**Pull request merged**